src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -1235,27 +1159,18 @@ private void WriteRedVisitor(bool genericResult)
                 if (nWritten > 0)
                     WriteLine();
                 nWritten++;
-                WriteComment(string.Format("<summary>Called when the visitor visits a {0} node.</summary>", node.Name));
-                WriteLine(
```

Comment:
yeah this mix of concatenation and format arguments is good to move away from.

Reply:
:)